script/DeployRehearsal5.s.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GnosisSafe} from "safe-contracts-v1.3.0/GnosisSafe.sol";
+import {GnosisSafeProxyFactory} from "safe-contracts-v1.3.0/proxies/GnosisSafeProxyFactory.sol";
+import {ModuleManager} from "safe-contracts-v1.3.0/base/ModuleManager.sol";
+import {OwnerManager} from "safe-contracts-v1.3.0/base/OwnerManager.sol";
+import {Enum} from "safe-contracts-v1.3.0/common/Enum.sol";
+
+import {SuperchainConfig} from "@eth-optimism-bedrock/src/L1/SuperchainConfig.sol";
+// import {DeputyGuardianModule} from "@eth-optimism-bedrock/src/Safe/DeputyGuardianModule.sol";
+
+import {ProxyAdmin} from "@eth-optimism-bedrock/src/universal/ProxyAdmin.sol";
+import {Proxy} from "@eth-optimism-bedrock/src/universal/Proxy.sol";
+import {EIP1967Helper} from "@eth-optimism-bedrock/test/mocks/EIP1967Helper.sol";
+
+import {Script} from "forge-std/Script.sol";
+
+import {console2 as console} from "forge-std/console2.sol";
+
+contract DeployRehearsal5 is Script {
+    GnosisSafe councilSafe;
+    GnosisSafe guardianSafe;
+    SuperchainConfig superchainConfigProxy;
+    address dummyDeputyGuardianModule = vm.envAddress("DUMMY_DEPUTY_GUARDIAN_MODULE");
+
+    address internal constant SENTINEL_OWNERS = address(0x1);
+
+    /// @notice The name of the script, used to ensure the right deploy artifacts
+    ///         are used.
+    function name() public pure returns (string memory name_) {
+        name_ = "DeployRehearsal5";
+    }
+
+    function setUp() public {
+        councilSafe = GnosisSafe(payable(vm.envAddress("COUNCIL_SAFE")));
+
+        console.log("Deploying from %s", name());
+        console.log("Deployment context: %s", vm.envOr("DEPLOYMENT_CONTEXT", string("Deployment context not set")));
+    }
+
+    function run() public {
+        console.log("Deploying contracts for rehearsal");
+        vm.startBroadcast();
+        deployGuardianSafe();
+        deploySuperchainConfigProxy();
+        vm.stopBroadcast();
+    }
+
+    function deployGuardianSafe() public {
+        address[] memory owners = new address[](1);
+        owners[0] = msg.sender;
```

Comment:
tangential, non-blocker: I like [solarray](https://github.com/emo-eth/solarray) for nicer array UX in tests/scripts. Might be nice to move towards this moving forward

Reply:
Noted :)